.vib/flink/goss/flink.yaml

```
@@ -0,0 +1,15 @@
+command:
+  check-flink-jobmanager-run:
+    exec: timeout --preserve-status 5 jobmanager.sh start-foreground
+    timeout: 8000
+    exit-status: 143
+    stdout:
+      - Rest endpoint listening
+      - Web frontend listening
+file:
+  /opt/bitnami/flink/plugins/s3-fs-base/flink-s3-fs-base-{{ .Env.APP_VERSION }}.jar:
+    exists: true
+  /opt/bitnami/flink/plugins/s3-fs-presto/flink-s3-fs-presto-{{ .Env.APP_VERSION }}.jar:
+    exists: true
+  /opt/bitnami/flink/plugins/s3-fs-hadoop/flink-s3-fs-hadoop-{{ .Env.APP_VERSION }}.jar:
+    exists: true
```

I would suggest adding the `filetype` entry here:
```suggestion
  /opt/bitnami/flink/plugins/s3-fs-base/flink-s3-fs-base-{{ .Env.APP_VERSION }}.jar:
    exists: true
    filetype: file
  /opt/bitnami/flink/plugins/s3-fs-presto/flink-s3-fs-presto-{{ .Env.APP_VERSION }}.jar:
    exists: true
    filetype: file
  /opt/bitnami/flink/plugins/s3-fs-hadoop/flink-s3-fs-hadoop-{{ .Env.APP_VERSION }}.jar:
    exists: true
    filetype: file
```